web/src/redux/apis/Test.api.ts

```diff
@@ -38,7 +38,9 @@ const TestAPI = createApi({
       query: id => `/tests/${id}`,
       providesTags: result => [{type: 'Test', id: result?.testId}],
     }),
-
+    deleteTestById: build.mutation<ITest, string>({
```

Comment:
I think we don't have to run the refetch, we can just invalidate the tag here that is going to automatically trigger the refetch of the entire list.

Reply:
makes sense. missed this one. but done